admin-dev/themes/new-theme/js/pages/order/view.js

```diff
@@ -55,7 +55,12 @@ $(() => {
   initAddCartRuleFormHandler();
   initAddProductFormHandler();
   initChangeAddressFormHandler();
-  
+  initHookTabs();
+
+  function initHookTabs() {
```

Comment:
I think you should use a comment if the goal is to explain what your chain does,  exept if your function is used elsewhere to avoid code duplication

Reply:
The thing is everywhere else in the file we have single use functions `initAddCartRuleFormHandler` `handlePaymentDetailsToggle` And as they say "in rome do as the romans do" so I don't want to start a new style in this file for such a small change It sure is not the best and this could justify to refacto it but it's not the purpose of this PR